librz/core/tui/visual.c

```diff
@@ -1987,7 +1987,7 @@ RZ_IPI void rz_core_visual_browse(RzCore *core, const char *input) {
 			rz_debug_switch_to_first_thread(core->dbg);
 			break;
 		case 'b':
-			rz_core_cmd0(core, "s $(afb~...)");
+			rz_core_seek_first_bb(core);
```

Comment:
It's more complicated than that - `afb~...` will allow you to select the particular basic block (just try it - you can scroll or type to search), so just going to the first one isn't enough.

Reply:
Oh, my bad. Will try to replicate it.